web/styles/subscriptions.css

```diff
@@ -1421,11 +1421,6 @@ div.settings-radio-input-parent {
     }
 }
 
-/* Note that this block has settings_page CSS as well, and thus needs
```

Comment:
There should probably be a note left behind here about coordinating with settings.css--probably parallel to the note that you've put there in this move.

Reply:
done!